be/src/olap/memtable.h

```diff
@@ -56,7 +61,13 @@ class MemTable {
 
     RowCursorComparator _row_comparator;
     std::unique_ptr<MemTracker> _mem_tracker;
-    std::unique_ptr<MemPool> _mem_pool;
+    // This is a buffer, to hold the memory referenced by the rows that have not
+    // been inserted into the SkipList
+    std::unique_ptr<MemPool> _tmp_mem_pool;
```

Comment:
_buffer_pool may be better

Reply:
OK, I will change it to `_buffer_mem_pool`